.travis.yml

```diff
@@ -22,4 +22,5 @@ install:
 
 script:
   - ulimit -n 4096
-  - powershell -File build/travis.ps1
\ No newline at end of file
+  - sudo powershell -c "Install-Module PowerShellGet -Force"
+  - powershell -File build/travis.ps1
```

Comment:
Does -File correctly report exit codes now?  At one time it didn't.

Reply:
hmm not sure, I was just adding the install-module above it. Wasn't really focusing on the travis build script itself.

Reply:
Just tried `pwsh` in Windows 10:
```

Name                           Value
----                           -----
PSVersion                      6.0.1
PSEdition                      Core
GitCommitId                    v6.0.1-289-g18d0a0bcbd436b11611ec13f5b2b4b290e58b542
OS                             Microsoft Windows 10.0.16299
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

Writing `bad.ps1` as:
```powershell
throw "Bad!"
```
Gives:
```
PS1> pwsh -File .\bad.ps1
Bad!
At C:\Users\roholt\bad.ps1:1 char:1
+ throw "Bad!"
+ ~~~~~~~~~~~~
+ CategoryInfo          : OperationStopped: (Bad!:String) [], RuntimeException
+ FullyQualifiedErrorId : Bad!

PS1> $LASTEXITCODE
1
```

Changing `bad.ps1` to:
```powershell
exit 1234
```
gives me:
```
PS1> pwsh -File .\bad.ps1

PS1> $LASTEXITCODE
1234
```

Reply:
huh. Interesting.

Reply:
Cool.  Looks like it got fixed somewhere along the way. I just tested in 5.1 and it works there too.  See Chris Oldwood's comment on this blog post - https://blogs.msdn.microsoft.com/powershell/2006/10/14/windows-powershell-exit-codes/

That is the behavior I remember.

Reply:
Good to know! Thanks for that, @rkeithhill.